hieradata/node/router-backend-1.router.staging.publishing.service.gov.uk.yaml

```diff
@@ -1,6 +1,6 @@
 govuk_env_sync::tasks:
   "push_router_daily":
-    ensure: "present"
```

Comment:
While you may have changed this based on my first comment. You may be able to just remove this hieradata, as I believe the host may no longer exist in Staging?

Reply:
Still there somehow: https://alert.staging.publishing.service.gov.uk/cgi-bin/icinga/status.cgi?host=all&type=detail&servicestatustypes=16&hoststatustypes=3&serviceprops=2097162

Reply:
I'm not sure what that link is meant to show, but at least looking at the list of UP hosts in Staging, I can't see router-backend-1.

Reply:
actually the alert is set up here: https://github.com/alphagov/govuk-puppet/blob/a2c2fe03001c93b802b0912f5b635310d4321005/hieradata/class/backup.yaml.

I will keep the 2 stage process of setting absent and then delete the hiera stuff.

Reply:
Sure, providing we're agreed that the machine doesn't exist, and there's a plan to eventually remove this entire file, then I'm happy.